mongodbforms/documentoptions.py

```diff
@@ -352,15 +369,21 @@ def __iter__(self):
     def __len__(self):
         return self._meta.__len__()
 
+    def __cmp__(self, other):
```

Comment:
In reality it's `__hash__` that I need to define to fix this line of django admin code:

https://github.com/django/django/blob/master/django/db/models/deletion.py#L58

Here's python's doc on why I also need to implement `__cmp__`:

> If a class does not define a __cmp__() or __eq__() method it should not define a __hash__() operation either; if it defines __cmp__() or __eq__() but not __hash__(), its instances will not be usable in hashed collections. If a class defines mutable objects and implements a __cmp__() or __eq__() method, it should not implement __hash__(), since hashable collection implementations require that a object’s hash value is immutable (if the object’s hash value changes, it will be in the wrong hash bucket).